packages/vscode-dependencies-validation/package.json

```diff
@@ -20,13 +20,18 @@
     "ci": "npm-run-all clean compile bundle",
     "clean": "rimraf ./dist *.vsix",
     "compile": "tsc -p ./",
+    "watch": "tsc -p ./ --watch",
     "test": "mocha",
     "coverage": "nyc mocha",
     "bundle": "webpack --mode production",
     "package": "node ./scripts/package-vsix.js"
   },
   "dependencies": {
-    "@sap-devx/npm-dependencies-validation": "file:../npm-dependencies-validation"
+    "@sap-devx/npm-dependencies-validation": "file:../npm-dependencies-validation",
```

Comment:
use package name, not relative link. you don't need to publish it to npm first
the pnpm/yarn/mono repo manager should handle these links

Reply:
Done